packages/passport-interface/src/RequestTypes.ts

```diff
@@ -178,6 +198,72 @@ export type CheckTicketInResponseValue = undefined;
  */
 export type CheckTicketInError = TicketError;
 
+/**
+ * A particular 'superuser' ticket-holder can request to check in
+ * another ticket that belongs to the same event, by referencing the ID
+ * of the ticket.
+ */
+export interface CheckTicketInByIdRequest {
+  /**
+   * A semaphore signature from the checker, used by the server to
+   * determine whether the checker has the required permissions
+   * to check this ticket in.
+   */
+  checkerProof: SerializedPCD<SemaphoreSignaturePCD>;
+
+  /**
+   * The ticket ID to attempt to check in.
+   */
+  ticketId: string;
+}
+
+/**
+ * On the happy path, {@link CheckTicketInByIdRequest} has nothing to say and
+ * just succeeds.
+ */
+export type CheckTicketInByIdResponseValue = undefined;
+
+/**
+ * A {@link CheckTicketInByIdRequest} can fail for a number of reasons.
+ */
+export type CheckTicketInByIdError = TicketError;
+
+/**
+ * When verifying scanned PCDs, we want to check with the server, which
+ * knows about public keys when the client does not.
+ */
+export interface VerifyTicketRequest {
+  /**
+   * A PCD to verify. JSON-encoded serialized PCD.
+   */
+  pcd: string;
+}
+
+export const enum SpecialTicketCategory {
+  Devconnect23 = "Devconnect23",
+  Zuzalu23 = "Zuzalu23",
+  Zuconnect23 = "Zuconnect23"
+}
+
+/**
+ * Result of verification, and name of the public key if recognized.
+ */
+export type VerifyTicketResponseValue =
+  | {
+      verified: true;
+      verifiedTicket: false;
```

Comment:
this type is confusing to me - what situation does this represent?

Reply:
The distinction between having verified the PCD as having a matching proof and claim, and having verified the ticket as being a real ticket. I'll make the terms different.